### ISSUE TITLE
Designs: Backcountry/PRDT inventory management

### ISSUE THREAD

**Issue by meyerdarcie** [bcgov/reserve-rec-public#25]
#### Description:

This ticket is to build out designs for backcountry inventory management. Note: designs will reflect additional offerings - broader than just backcountry. 

#### Acceptance Criteria:

- [x] Designs are in figma and linked here 
- [x] User journeys follows the figjam workflow and the user story map (linked below)

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Wireframe(s):**
[Figma - PRDT Sketches](https://www.figma.com/design/1QHd1cWiRqeC0Ua0E9nfhr/PRDT?node-id=201-956&t=uYGG3TyM1gPQSH4m-1)

**Workflows:**
[Backcountry booking flow](https://www.figma.com/board/HNsft6y1lXjuGF2mh1Icv7/Backcountry-Booking-Flow?node-id=0-1&node-type=canvas&t=HprvJqvSG6r0tW2d-0)

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by meyerdarcie** [comment]
@pavelcreates FYI on inventory management ticket

**Comment by Dianadec** [comment]
Pavel to continue when he is back from vacay. Adding to blocked

**Comment by Dianadec** [comment]
Pavel to split this - designs that are done can be part of the discovery.
Inventory management (selecting park and site and adding to cart) as part of this ticket.
New ticket for permit applications section.

Sending to Done; @pavelcreates  to create new ticket.

**Comment by LindsayMacfarlane** [comment]
Thank you for all your work on these designs @pavelcreates ! Looking forward to the next stages :)
